queries/php/locals.scm

```diff
@@ -2,21 +2,42 @@
 ;-------
 
 (class_declaration) @scope
-(property_declaration) @scope
-(method_declaration) @scope
-(function_definition) @scope
 (while_statement) @scope
 (foreach_statement) @scope
 (if_statement) @scope
 (try_statement) @scope
+(method_declaration
+  name: (name) @definition.method) @scope
+(function_definition
+  name: (name) @definition.function) @scope
 
 ; Definitions
 ;------------
 
-(variable_name
-  (name) @definition.var)
+(simple_parameter
+  (variable_name
+    (name) @definition.var))
+
+(anonymous_function_use_clause
+  (variable_name
+    (name) @definition.var))
+
+(assignment_expression
+  left: (variable_name
+    (name) @definition.var))
+
+(property_declaration
+  (variable_name
+    (name) @definition.field))
+
+(namespace_use_clause
+  (qualified_name
+    (name) @definition.class))
+
+(class_declaration
+  name: (name) @definition.class)
 
 ; References
 ;------------
 
-(variable_name) @reference
```

Comment:
Why ?

Reply:
(variable_name) contains "$" and (name), and class names, function names, etc. do not contain "$".
"$", if present, will not match all of the names except (variable_name), so I changed (variable_name) to (name).